### ISSUE TITLE
[v4] [Android] [Talkback] [Accessible] Cannot select other components on screen after moving Bottomsheet

### ISSUE THREAD

**Issue by vannam-tran** [gorhom/react-native-scroll-bottom-sheet#1914]
# Bug

## Environment info

| Library                         | Version |
| ------------------------------- | ------- |
| @gorhom/bottom-sheet            | 4.6.3   |
| react-native                    | 0.7.4   |
| react-native-reanimated         | 3.12.1   |
| react-native-gesture-handler    | 2.12.1   |

## Steps To Reproduce

1. Enable Talkback on Android then move to the Bottomsheet
2. Try to move to full screen or even just a little move only then cannot tap any elements on screen anymore.
3. User can move the Bottomsheet back to mini or medium mode but cannot tap any elements on screen (even elements inside the bottomsheet).

Describe what you expected to happen:

1. User can tap to any elements on screen after moving the bottomsheet

**Comment by vannam-tran** [comment]
The issue comes from `react-native-gesture-handler` library. It seems wrong in implementation of Hover feature that make this conflicted with scroll event. 
https://github.com/software-mansion/react-native-gesture-handler/commit/16a266e3af637c24cf690a3c76995b178d40f453

After editing the override function then Talkback now can work again after moving bottomsheet (I ignored Hover feature).
node_modules/react-native-gesture-handler/android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerRootView.kt
```
override fun dispatchGenericMotionEvent(event: MotionEvent) =
    super.dispatchGenericMotionEvent(event)
```

This bug should be reported to `react-native-gesture-handler` library!
